src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchViewModel.cs

```diff
@@ -913,5 +916,12 @@ public void ExecuteSelected()
             SearchResults[SelectedIndex].Model.Execute();
         }
 
+        private void OnPackageAddedOrRemoved(Package _)
+        {
+            foreach (var sr in SearchResults)
+            {
+                sr.CanInstall = CanInstallPackage(sr.Model.Name);
```

Comment:
O every added/removed package we will update the CanInstall flag for each searchElement